### PR TITLE
[nodejs][09.message routing] Fix typos and improve formatting

### PR DESCRIPTION
**Pull request by joseques**
## Proposed Changes
- Minor changes to typos and minimal structural changes.

### CODE REVIEW

File: samples/javascript_nodejs/09.message-routing/README.md [hunk 1]
```diff
@@ -11,12 +11,12 @@ This samples shows how to:
...
   git clone https://github.com/microsoft/botbuilder-samples.git
   ```
-- In a terminal, navigate to javascript_nodejs/09.message-routing
+- In a terminal, navigate to `javascript_nodejs/09.message-routing`
   ```bash
   cd javascript_nodejs/09.message-routing
   ```
-  - [Optional] Update the .env file under samples/javascript_nodejs/09.message-routing with your botFileSecret
-    For Azure Bot Service bots, you can find the botFileSecret under application settings.
+  - [Optional] Update the `.env` file under `samples/javascript_nodejs/09.message-routing` with your `botFileSecret`
```
Comment (VictorGrycuk) [inline samples/javascript_nodejs/09.message-routing/README.md:18]: This line has an indentation which is not found in other readme's.

**Review by VictorGrycuk — CHANGES_REQUESTED** [review]
[Line 24](https://github.com/southworkscom/BotBuilder-Samples/pull/96/files#diff-df67f087e0a1f303872760f62d7c58fcR24) leads to a non-existent file

File: samples/javascript_nodejs/09.message-routing/README.md (hunk 1 above)
Comment (joseques) [reply samples/javascript_nodejs/09.message-routing/README.md:18]: @VictorGrycuk 
Line 18 fixed.
For Line 24 see Issue #95

**Review by VictorGrycuk — APPROVED** [review]
Nice

**Review by dfavretto — CHANGES_REQUESTED** [review]
Fix typo in [line 8](https://github.com/southworkscom/BotBuilder-Samples/pull/96/files#diff-ec7a229943e151a666b335d676aba62cL8) in `dialogs/greeting/greeting.js`

**Comment by joseques** [comment]
Fixed @dfavretto

**Review by dfavretto — APPROVED** [review]

**Review by dfavretto — CHANGES_REQUESTED** [review]
The `Readme.md` file is missing the `Deploy this bot to Azure` section. Please copy and adapt it from its [netcore counterpart sample](https://github.com/Microsoft/BotBuilder-Samples/blame/3ebca19a901ecfe3433b3660f0ed05f517a937d2/samples/csharp_dotnetcore/09.message-routing/readme.md#L24).

**Comment by joseques** [comment]
Sample doesn't has any existent deployment files @dfavretto

**Review by dfavretto — APPROVED** [review]

**Comment by dfavretto** [comment]
Closed due to Microsoft#740

**Pull request closed without merging**